homeassistant/components/geocaching/translations/nl.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Don't add additional languages, this can be translated via the Lokalize platform once it has been merged.

Reply:
OK!